Scryfall Tools/limited_pools.py

```diff
@@ -4,25 +4,116 @@
 
 
 def get_pack(set_code):
-    # Get a rare or mythic (~14% chance of Mythic)
-    if random.randint(0, 7) == 7:
-        rare_card = st.get_random_card(f'set:{set_code} r=m in:booster')
-    else:
-        rare_card = st.get_random_card(f'set:{set_code} r=r in:booster')
+    def get_cmr_pack(set_code):
+        # Commander Legends pack distribution        
+        pack = []
+        
+        # Get 1 foil
+        # Randomly determine rarity of foil
+        foil_rng = random.randint(0, 119)
+        if(foil_rng == 119):
```

Comment:
Python tries to avoid unnecessary brackets around statements:

```suggestion
        if foil_rng == 119:
```